CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ Guidelines](https://opensource.google/conduct/).
 2. Run tests with `go test ./...`
 3. You may need to run "live" tests that talk to a real endpoint; to do so, run
    `go test -v ./genai/...` passing it your API key with the `-apikey` flag
-   and a model name flag like `-model gemini-pro`
```

Comment:
is this seeping in from the other PR we wanted to wait on?

Reply:
Oops. I think I fixed it.